Shared/sdk/SharedUtil.Misc.h

```diff
@@ -740,6 +742,17 @@ namespace SharedUtil
         void    Insert(const SString& strInCmd, int iValue);                                                // Insert a key int value
         void    Insert(const SString& strInCmd, const SString& strInValue);                                 // Insert a key string value
         bool    Contains(const SString& strInCmd) const;                                                    // Test if key exists
+
+        std::string Get(const char*& inCmd) const noexcept;
+        bool        Get(const char*& inCmd, std::string& out, const char* defaultValue = "") const noexcept;
+        bool        Get(const char*& inCmd, std::vector<std::string>& outList) const noexcept;
+        bool        Get(const char*& inCmd, int& value, int defaultValue = 0) const noexcept;
+
+        std::string Get(const std::string& inCmd) const noexcept;
+        bool Get(const std::string& inCmd, std::string& out, const char* defaultValue = "") const noexcept;
+        bool Get(const std::string& inCmd, std::vector<std::string>& outList) const noexcept;
+        bool Get(const std::string& inCmd, int& value, int defaultValue = 0) const noexcept;
```

Comment:
All of these methods can throw an exception. So please, remove `noexcept` specifier.

Reply:
None of these can throw an exception.

Reply:
Only the vector one can throw an exception. I will change that to non noexcept.
Every other method wont throw an exception